modules/public/logic.py

```diff
@@ -142,6 +186,13 @@ def get_pages_details(url, links):
 
 
 def convert_limit_offset_to_size_page(url, link):
+    """Convert navigation link from offst/limit to size/page
```

Comment:
`s/offst/offset/`

Reply:
Could you make this one a bit more descriptive? Give examples of input and output URLs maybe?